### PR TITLE
Use projects package.json instead of plugins

### PR DESCRIPTION
**Pull request by victorb**
This closes issue #4 where the plugins `package.json` version is being used instead of the projects package.json

### CODE REVIEW

**Comment by markmarijnissen** [comment]
thanks, wonderful!

**Comment by markmarijnissen** [comment]
and published to npm!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "webpack-cordova-plugin",
-  "version": "0.1.5",
+  "name": "webpack-cordova-plugin-fork",
```
Comment (TCMiranda) [inline package.json:2]: Hey, thats wrong isn't it??

Comment (victorb) [inline package.json:2]: Yes it is indeed! Before this PR was merged, I needed this to be in NPM but accidentally made my changes to the same branch as I used for this PR. Not sure how @markmarijnissen could publish new version to NPM.

Sorry for this, totally my mistake.

Comment (TCMiranda) [inline package.json:2]: No problem! I just found it weird that it has been published also
Thanks though!

**Pull request merged**